src/lexer.coffee

```diff
@@ -121,7 +121,11 @@ exports.Lexer = class Lexer
         'PROPERTY'
       else
         'IDENTIFIER'
-
+    
+    if tag is 'IDENTIFIER' and @seenFor and id is 'from'
```

Comment:
Fair enough. It's still a bit weird to have this part disjointed from the others.